functions/uninstall.go

```diff
@@ -14,12 +15,16 @@ import (
 	"github.com/gravitl/netclient/wireguard"
 	"github.com/gravitl/netmaker/logger"
 	"github.com/gravitl/netmaker/models"
+	"golang.org/x/exp/slog"
 )
 
 // Uninstall - uninstalls networks from client
 func Uninstall() ([]error, error) {
 	allfaults := []error{}
 	var err error
+	if runtime.GOOS == "windows" {
```

Comment:
Should this continue processing or exit? If exit, probably want to move check prior to call to cleanup so that server is informed that host/node is leaving and /etc/hosts entries are cleaned up.

Reply:
done.